dev-tex/classico/Manifest

```diff
@@ -0,0 +1 @@
+DIST classico.tds.zip 1312858 BLAKE2B 3348ae972806f19994e21ea68d68a5d8f775a1fcb5736dc4fa077e7e266ef48ba1dc6e0c449966aeace708d505d8f72c255e8f916799b28be3e3a866767f589e SHA512 c56f830a98a9c164b9483ddb3562270c35651ac6b7b87b099be1df0e0c0db5c974ec2edc80c33b775af6a4bc09181124c37c24cb4df26979cddd152816c8d263
```

Comment:
The filename should really contain version somehow. Otherwise, you won't be able to bump it without causing distfile collisions.

Reply:
This is a problem with CTAN which doesn't list version numbers for some reason. Some packages are available and versioned on the TeXlive server but not this one.

Some packages use `mirror://gentoo/${P}.zip` as the source URI. Maybe this is the only option for this package?

Reply:
You can use `-> newname` in SRC_URI to rename the file. While I guess it's not really necessary for the first version, I think it might be reasonable to put version there somehow. Preferably in a way consistent with how those 'good' packages do, be it `${P}.tds.zip` or something similar.